library/std/src/os/openbsd/fs.rs

```diff
@@ -1,6 +1,7 @@
 #![stable(feature = "metadata_ext", since = "1.1.0")]
 
 use crate::fs::Metadata;
+use crate::sealed::Sealed;
```

Comment:
missing bound on `MetadataExt`?

Reply:
Thanks! Not sure why the one got missed.